torchtune/modules/peft/lora.py

```diff
@@ -65,7 +65,7 @@ def __init__(
         self.register_parameter(
             "bias", nn.Parameter(bias) if bias is not None else None
         )
-        self.dropout = nn.Dropout(p=dropout)
+        self.dropout = nn.Dropout(p=dropout) if dropout > 0.0 else nn.Identity()
```

Comment:
Did you try inplace or no?

Reply:
i didnt, but i googled a bit about it and saw that people reporting issues with backward computation. In my experience, pytorch doesnt like inplace computations, because it needs the input preserved for gradient calculation.